From: Christoph Hellwig <hch@lst.de>
Date: Tue, 1 Nov 2016 07:40:07 -0600
Subject: block: don't use REQ_SYNC in the READ_SYNC definition
References: bsc#1020989,FATE#322379
Git-commit: 6f6b29171a192e84b666c816e49d2175afbbb09f
Patch-Mainline: v4.10-rc1

Reads are synchronous per definition, don't add another flag for it.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jens Axboe <axboe@fb.com>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 include/linux/fs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/fs.h b/include/linux/fs.h
index 9cfc9ec..23e0b3c 100644
--- a/include/linux/fs.h
+++ b/include/linux/fs.h
@@ -193,7 +193,7 @@ typedef int (dio_iodone_t)(struct kiocb *iocb, loff_t offset,
 #define READ			REQ_OP_READ
 #define WRITE			REQ_OP_WRITE
 
-#define READ_SYNC		REQ_SYNC
+#define READ_SYNC		0
 #define WRITE_SYNC		(REQ_SYNC | REQ_NOIDLE)
 #define WRITE_ODIRECT		REQ_SYNC
 #define WRITE_FLUSH		(REQ_SYNC | REQ_NOIDLE | REQ_PREFLUSH)
-- 
1.8.5.6

